--- operator/cmd/mesh/upgrade.go
+++ operator/cmd/mesh/upgrade.go
@@ -62,7 +62,8 @@ const (

 type upgradeArgs struct {
	// inFilenames is an array of paths to the input IstioOperator CR files.
-	inFilenames []string
+	inFilenames           []string
+	currentVersionProfile string
	// kubeConfigPath is the path to kube config file.
	kubeConfigPath string
	// context is the cluster context in the kube config.
@@ -84,6 +85,8 @@ type upgradeArgs struct {
 func addUpgradeFlags(cmd *cobra.Command, args *upgradeArgs) {
	cmd.PersistentFlags().StringSliceVarP(&args.inFilenames, "filename",
		"f", nil, filenameFlagHelpStr)
+	cmd.PersistentFlags().StringVarP(&args.currentVersionProfile, "currentVersionProfile", "",
+	"", "Path to existing clusters IstioOperator profile")
	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig",
		"c", "", KubeConfigFlagHelpStr)
	cmd.PersistentFlags().StringVar(&args.context, "context", "",
@@ -186,7 +189,11 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l clog.Logger) (err error) {
	// Read the current installation's profile IOP yaml to check the changed profile settings between versions.
	currentSets := args.set
	if currentVersion != "" {
-		currentSets = append(currentSets, "installPackagePath="+releaseURLFromVersion(currentVersion))
+		if args.currentVersionProfile == "" {
+			currentSets = append(currentSets, "installPackagePath="+releaseURLFromVersion(currentVersion))
+		} else {
+			currentSets = append(currentSets, "installPackagePath="+args.currentVersionProfile)
+		}
	}
	profile := targetIOP.Spec.Profile
	if profile == "" {
