diff --git a/common/scripts/run.sh b/common/scripts/run.sh
index 4a2ebc94c6..86bb67ef77 100755
--- common/scripts/run.sh
+++ common/scripts/run.sh
@@ -21,7 +21,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.

-set -e
+set -x

 WD=$(dirname "$0")
 WD=$(cd "$WD"; pwd)
@@ -45,7 +45,6 @@ read -ra DOCKER_RUN_OPTIONS <<< "${DOCKER_RUN_OPTIONS:-}"
 "${CONTAINER_CLI}" run \
     --rm \
     "${DOCKER_RUN_OPTIONS[@]}" \
-    --init \
     --sig-proxy=true \
     --cap-add=SYS_ADMIN \
     ${DOCKER_SOCKET_MOUNT:--v /var/run/docker.sock:/var/run/docker.sock} \
